src/404.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: minimal.html
+pageTitle: Page not found
+---
+
+<div class="site-grid-404">
+  <div class="header-404-wrapper">
+   <h2>Oops!</h2>
+   <h4>We can't seem to find the page you're looking for.</h4>
+  </div>
+
+  <a role="button" href="/" class="button-home">
```

Comment:
```suggestion
  <a href="/" class="button-home">
```

Just a link; the `role` attribute has other affects and isn't necessary if you only want your link to _look_ button-ish.